angular/projects/spark-angular/src/lib/components/sprk-accordion-item/sprk-accordion-item.component.spec.ts

```diff
@@ -317,23 +319,24 @@ describe('SprkAccordionItemComponent', () => {
     ).toEqual('#exclamation');
   });
 
-  it('should emit open and closed events when toggled', (done) => {
+  it('should emit open event when toggled open', (done) => {
     let openEventEmitted = false;
-    let closedEventEmitted = false;
-
     component.openedEvent.subscribe((g) => {
       openEventEmitted = true;
       done();
```

Comment:
New jest doesnt allow for done() to be called twice in one test so I refactored all the tests that did this and separated out the tests into two distinct tests checking either the open or closed event.